EToCalculator.ts

```diff
@@ -0,0 +1,62 @@
+import { EToData } from "./types";
```

Comment:
Could this be packaged together with the Adjustment Option code in `EToAdjustmentMethod.ts`

Reply:
Good idea